nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("welcome to nodejs week3 homework"));
+app.get("/calculator/add", (req, res) => {
+    const { first, second } = req.query;
+    res.send(`<h1>Addition of given numbers :  ${parseInt(first) + parseInt (second)}</h1>`);
+});
+app.get("/calculator/multiply", (req, res) => {
+    const { first, second } = req.query;
+    res.send(`<h1>Multiplication of given numbers : ${parseInt(first) * parseInt(second)}</h1>`);
+});
+app.get("/calculator/subtraction", (req, res) => {
+    const { first, second } = req.query;
+    res.send(`<h1>Substraction of given numbers : ${parseInt(first) - parseInt(second)}</h1>`);
+});
+app.get("/calculator/division", (req, res) => {
+    const { first, second } = req.query;
+    res.send(`<h1>Division of given numbers : ${parseInt(first) / parseInt(second)}</h1>`);
+});
```

Comment:
There is an additional use case in the homework:
What if there not only two parameters in the query? There could be a infinite number of query parameters